### DISCUSSION TITLE
SFDX Environment Variable to override the directory where .sfdx/* is stored

### DISCUSSION THREAD

**Discussion by mjyocca** [forcedotcom/cli#2373] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm trying to use a lambda container function to authenticate with our devhub however I'm running into issues as the only directory during lambda execution that is available to write to is the tmp directory.

Receiving: `SfdxError: ENOENT: no such file or directory, open '/home/sbx_user1051/.sfdx/sfdx.log'`

Attempted to get around the error by assigning the $HOME to `/tmp`, error went away however attempting to sfdx auth:jwt:grant hangs and is unresponsive. 

It would be great if there was an option to tell the cli to only store .sfdx files in a particular directory.

[a comment by an automated account is not shown]

**Comment by daraul** [comment]
This would be useful for keeping separate one's work, and personal organizations.
